setup/docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
     image: redis:3.0-alpine
     restart: always
   postgres:
-    image: postgres:9.5.6-alpine
+    image: postgres:9.5.16-alpine
```

Comment:
Does Postgres publish `postgres:9.5` image that tracks latest patch version?

Reply:
Good point.  Yes it does.  1 sec, will update. :smile: